examples/orderbook/create-listing-with-nextjs/src/app/utils/setupOrderbook.ts

```diff
@@ -0,0 +1,8 @@
+import { orderbook } from "@imtbl/sdk";
+import { Environment } from "@imtbl/config";
```

Comment:
This should be imported from @imtbl/sdk rather than @imtbl/config. Yarn currently is quite lax with imports unfortunately and allows this behaviour.